proptest-macro/src/property_test/codegen/mod.rs

```diff
@@ -1,9 +1,13 @@
 use proc_macro2::TokenStream;
 use quote::{quote, ToTokens};
-use syn::{parse_str, spanned::Spanned, Attribute, Ident, ItemFn, PatType};
+use syn::{parse_str, spanned::Spanned, Attribute, Ident, ItemFn};
 
-use super::{options::Options, utils::strip_args};
+use super::{
+    options::Options,
+    utils::{strip_args, Argument},
+};
 
+mod arbitrary;
```

Comment:
i dont think this file has been committed yet -- which also exposes gaps in our CI which likely isn't building this new crate

Reply:
Good spot - let's get this in CI